### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by odow**
Part of https://github.com/jump-dev/JuMP.jl/pull/3309

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,99 @@
-# COIN-OR SemiDefinite Programming Interface (CSDP.jl)
-
 ![](https://www.coin-or.org/wordpress/wp-content/uploads/2014/08/COINOR.png)
+s
```
Comment (blegat) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,99 @@
...
 
 The original algorithm is described by B. Borchers.
 _CSDP, A C Library for Semidefinite Programming_.
 Optimization Methods and Software 11(1):613-623, 1999.
 DOI [10.1080/10556789908805765](http://dx.doi.org/10.1080/10556789908805765).
 [Preprint](http://euler.nmt.edu/~brian/csdppaper.pdf).
 
+## License
+
+`Clp.jl` is licensed under the [MIT License](https://github.com/jump-dev/CSDP.jl/blob/master/LICENSE.md).
```
Comment (blegat) [inline README.md]:
```suggestion
`CSDP.jl` is licensed under the [MIT License](https://github.com/jump-dev/CSDP.jl/blob/master/LICENSE.md).
```

File: README.md [hunk 3]
```diff
@@ -108,27 +157,4 @@ If the `printlevel` option is at least `1`, the following will be printed:
...
-`perturbobj`  | The `perturbobj` parameter determines whether the objective function will be perturbed to help deal with problems that have unbounded optimal solution sets. If `perturbobj` is `0`, then the objective will not be perturbed. If `perturbobj` is `1`, then the objective function will be perturbed by a default amount. Larger values of `perturbobj` (e.g. `100`) increase the size of the perturbation. This can be helpful in solving some difficult problems. | `1`            |
-`fastmode`    | The `fastmode` parameter determines whether or not CSDP will skip certain time consuming operations that slightly improve the accuracy of the solutions. If `fastmode` is set to `1`, then CSDP may be somewhat faster, but also somewhat less accurate | `0`            |
-`printlevel`  | The `printlevel` parameter determines how much debugging information is output. Use a `printlevel` of `0` for no output and a `printlevel` of `1` for normal output. Higher values of printlevel will generate more debugging output | `1`            |
+  * and the DIMACS error measures.s
```
Comment (blegat) [inline README.md]:
```suggestion
  * and the DIMACS error measures.
```

Comment (odow) [reply README.md]: Man I swear my `CTRL+S` is failing. Dropping `s` all over the place

**Review by blegat — APPROVED** [review]

**Pull request merged**